tensorbay/exception.py

```diff
@@ -8,17 +8,97 @@
 The class hierarchy for TensorBay custom exceptions is::
 
      +-- :class:`TensorBayException`
+         +-- :class:`TensorBayClientError`
+             +-- :class:`CommitStatusError`
+             +-- :class:`DatasetTypeError`
+             +-- :class:`FrameError`
+             +-- :class:`ResponseError`
          +-- :class:`TensorBayOpenDatasetError`
              +-- :class:`NoFileError`
              +-- :class:`FileStructureError`
 
 """
 
+from requests.models import Response
+
 
 class TensorBayException(Exception):
     """This is the base class for TensorBay custom exceptions."""
 
 
+class TensorBayClientException(TensorBayException):
+    """This is the base class for custom exceptions in TensorBay client module."""
+
+
+class CommitStatusError(TensorBayClientException):
+    """This class defines the exception for illegal commit status.
+
+    Arguments:
+        message: The error message.
+
+    """
+
+    def __init__(self, message: str) -> None:
```

Comment:
is that possible to build message in `__str__` rather than pass the message directly in to the exceptions. Just like `DatasetTypeError`